### PR TITLE
Developers and Games

### PR DESCRIPTION
**Pull request by websupergirl**
Moved submit info from games to developers

### CODE REVIEW

**Comment by websupergirl** [comment]
for https://github.com/indiegamecon/indiegamecon-gatsby/issues/111

File: src/markdown/developers.md [hunk 1]
```diff
@@ -24,3 +24,54 @@ https://www.glimmer.tech
 #### Bitforest
 
 http://www.bitforest.co
+
+---
+
+### [Submit Your Game!]
```
Comment (mckelveygreg) [inline src/markdown/developers.md:30]: remove these brackets, it is making an invalid markdown file, unless do you want to make this a link? then just add 
`### [Submit Your Game!](/games)`

**Review by mckelveygreg — CHANGES_REQUESTED** [review]
Either add a link to the games pages or remove the brackets and it should be fine. 
I left a snippet in my comment

**Comment by mckelveygreg** [comment]
fixed in #116

**Pull request closed without merging**